services/tas/tas-tests.tester.js

```diff
@@ -0,0 +1,8 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('tas tests').get('/github/tasdemo/axios.json').expectBadge({
+  label: 'tests',
+  message: Joi.string(),
```

Comment:
Can we make this more specific. This isn't really a useful test as an error message will match `Joi.string()` too. There are some validators in `services/test-validators.js` for testing this type of test summary badge: https://github.com/badges/shields/blob/6e803367e0f3c8e0cc4196a700af37fac1629f4d/services/test-validators.js#L135-L154

Also can we have a couple of tests for the behaviour in the 401 and 404 cases.

Reply:
added.